x/group/spec/03_msg_service.md

```diff
@@ -0,0 +1,104 @@
+<!--
```

Comment:
The SDK has a [SPEC-SPEC](https://github.com/cosmos/cosmos-sdk/blob/master/docs/spec/SPEC-SPEC.md), which recommends `03_messages.md` for this file. I propose we follow that (even though those are all service Msgs)